hack/license-header-check.sh

```diff
@@ -24,50 +24,50 @@ MISSING_LICENSE_FILES=()
 
 # Define the essential Apache 2.0 license lines
 LICENSE_LINES=(
-  "http://www.apache.org/licenses/LICENSE-2.0"
+    "http://www.apache.org/licenses/LICENSE-2.0"
 )
 
 # Find all .go and .sh files, excluding specified paths
 mapfile -t FILES < <(find "$ROOT_DIR" \
-  -type f \( -name "*.go" -o -name "*.sh" \) \
-  ! -path "$ROOT_DIR/vendor/*" \
-  ! -path "$ROOT_DIR/kubevirtci/*" \
-  ! -path "$ROOT_DIR/tools/*" \
-  ! -path "$ROOT_DIR/staging/*" \
-  ! -path "$ROOT_DIR/.bazeldnf/*" \
-  ! -path "$ROOT_DIR/pkg/handler-launcher-com/*" \
-  ! -path "$ROOT_DIR/pkg/hooks/*" \
-  ! -path "$ROOT_DIR/pkg/virt-handler/device-manager/deviceplugin/v1beta1/api.pb.go" \
-  ! -path "$ROOT_DIR/pkg/vsock/system/v1/system.pb.go" \
-  ! -name "generated_mock*.go")
+    -type f \( -name "*.go" -o -name "*.sh" \) \
+    ! -path "$ROOT_DIR/vendor/*" \
+    ! -path "$ROOT_DIR/kubevirtci/*" \
+    ! -path "$ROOT_DIR/tools/*" \
+    ! -path "$ROOT_DIR/staging/*" \
+    ! -path "$ROOT_DIR/.bazeldnf/*" \
+    ! -path "$ROOT_DIR/pkg/handler-launcher-com/*" \
+    ! -path "$ROOT_DIR/pkg/hooks/*" \
```

Comment:
There are files in `pkg/hooks` that should have the header. Can you check all the exclusions again to see if they can be made more granular?

Reply:
Thanks @0xFelix Dan said this pr is too big and we should start with adding this linter only to small sub directory as a start. I will open a new pr to add it only for pkg/monitoring and we will go from there